### PR TITLE
Remove external link tracker

### PR DESCRIPTION
**Pull request by boffbowsh**
The external link tracker is being removed, so this repo should no
longer push recommended links into the tracker’s whitelist.

### CODE REVIEW

**Comment by alext** [comment]
:+1: Looks good.  For safety, this should probably not be merged until https://github.com/alphagov/frontend/pull/885 has been deployed.

File: test/acceptance/rake_task_test.rb [hunk 1]
```diff
@@ -19,8 +19,7 @@ class RakeTaskTest < Test::Unit::TestCase
       )
       indexer.expects(:index).with([expected_recommended_link])
       indexer.expects(:remove).with([expected_deleted_link])
-      external_link_registerer.expects(:register_links)
-                              .with([expected_recommended_link])
+      external_link_registerer.expects(:register_links).never
```
Comment (jamiecobbett) [inline test/acceptance/rake_task_test.rb]: I think we can safely remove this and the stub itself now.

Comment (boffbowsh) [inline test/acceptance/rake_task_test.rb]: I kinda thought it'd be a good way of checking that it's not called at all.

**Comment by jamiecobbett** [comment]
Apart from one minor comment, this is good to merge :+1:

File: test/acceptance/rake_task_test.rb (hunk 1 above)
Comment (rboulton) [inline test/acceptance/rake_task_test.rb]: I don't think this we need to do that - ensuring that the ExternalLinkRegisterer class isn't defined in the code is a pretty thorough check of this, too.

**Comment by rboulton** [comment]
This is ready to be merged now - the relevant frontend PR has been deployed.

**Comment by rboulton** [comment]
Sorry - I should have been clearer.  This PR is no longer blocked from being merged, but I'm not yet merging it because I think it should be updated to remove all trace of `external_link_registerer`, including from the tests.

**Comment by rboulton** [comment]
:fireworks:

**Pull request merged**